src/System Application/App/Entity Text/src/EntityTextImpl.Codeunit.al

```diff
@@ -41,20 +51,26 @@ codeunit 2012 "Entity Text Impl."
 
     [NonDebuggable]
     procedure GenerateSuggestion(Facts: Dictionary of [Text, Text]; Tone: Enum "Entity Text Tone"; TextFormat: Enum "Entity Text Format"; TextEmphasis: Enum "Entity Text Emphasis"; CallerModuleInfo: ModuleInfo): Text
+    begin
+        exit(GenerateSuggestion(Facts, Tone, TextFormat, TextEmphasis, CallerModuleInfo, ''));
+    end;
+
+    [NonDebuggable]
```

Comment:
NonDebuggable is too wide in this case. It should only cover the methods that are sensitive. 
IsEnabled and CanSuggest should be debuggable, then from BuildPrompts and GenerateAndReviewCompletion it should be non-debuggable